### DISCUSSION TITLE
Feature Request: html/gwt cache warnings

### DISCUSSION THREAD

**Discussion by payne911** [BlueBoxWare/LibGDXPlugin#27] [General]
A few pitfalls when trying to target the HTML platform is that editing the GWT XML file in the `html` module will not be detected as a modification affecting the cache: one has to manually delete both the `build` and `war` folders to trigger the proper compilation of the change.

Apparently, a simple `gradlew clean` does not end up cleaning the `war` folder, so that's yet another pitfall.

What if there was a warning coming from this plugin whenever the GWT XML file is modified?

**Comment by BlueBoxWare** [comment]
That's a good idea. When I'm done with improving and fixing the parsing of JSON, I will look into that.

**Comment by BlueBoxWare** [comment]
I think I have it working. Can you try it out? Also, do you have a suggestion for what the popup should say? The current text (`*.gwt.xml files have been changed. The module needs a full rebuild.`) is not great.

https://github.com/BlueBoxWare/LibGDXPlugin/releases/download/1.23-pre/LibGDX.Plugin-1.23-pre.zip

**Reply by payne911** [reply]
Very sorry for not coming back to you. I'm very occupied with some personal matter, these days.

I had asked for others to check it out on the libGDX Discord but the only feedback I got was that you should come hang out in the server. (Yes, I know you're not specifically interested in that, and I linked them to [here](https://github.com/BlueBoxWare/LibGDXPlugin/issues/22#issuecomment-692317449).)

I now see you've actually pushed a release for it, so thanks for your efforts! :)
